config/index.js

```diff
@@ -7,8 +7,10 @@ module.exports = {
     port: process.env.PORT || 8001,
     logs: {
       console: true,
-      app: false,
-      error: false
+      app: 'logs/lev-web-ui.log',
+      error: false,
+      dateRotate: true,
+      maxFiles: 7,
```

Comment:
Is my understanding here, that `dateRotate` keeps the last 7 backups of a file after rotation?

In https://github.com/UKHomeOffice/lev-api/blob/main/log-rotation.conf I've not defined the parameter `rotate x` where x is some number. This is because each log is pushed into the ELK stack, and we are maintaining a stateless service.

I don't see a plan to persist these local log files into any persistent storage because of the above

Reply:
from the documentation of hmpo-logger, maxFiles keeps number of files you specify and its based on date. so it keeps the record for 7 days and rotate it. 

<img width="784" alt="Screenshot 2023-08-17 at 10 29 44" src="https://github.com/UKHomeOffice/lev-web-ui/assets/139459624/e7625c03-c40a-44a6-841e-b00c9dfac999">

Reply:
it would be good when you have time to come on call. What are we trying to achieve here, I just used the log rotation from hmpo-logger specified. however it sees like from your comment that we need something else for persistency of data. It would be great if you can explain that further

Reply:
It has been decided that we keep the rotation of file to maxFile=1 and dateRotate to be true. so we don't build up logs on a container and filebeat picks the log up from containers timely manner.